### PR TITLE
Security update, to 7.71.1

### PR DESCRIPTION
**Pull request by PallHaraldsson**

### CODE REVIEW

**Comment by giordano** [comment]
@aviks can we drop the certificate now that's in a different package?

**Comment by aviks** [comment]
Yes, I think so.

**Comment by PallHaraldsson** [comment]
I just thought the certificate should be updated too, and in sync. Feel free to merge as is, or tell me if I need to change anything. It seems it doesn't matter if it's merged with and dropped later.

**Comment by giordano** [comment]
> Feel free to merge as is, or tell me if I need to change anything.

Yes, remove the certificate please, it has its own package now

File: L/LibCURL/build_tarballs.jl [hunk 1]
```diff
@@ -53,8 +48,7 @@ platforms = supported_platforms()
 # The products that we will ensure are always built
 products = [
     LibraryProduct("libcurl", :libcurl),
-    ExecutableProduct("curl", :curl),
-    FileProduct("share/cacert.pem", :cacert)
+    ExecutableProduct("curl", :curl)
```
Comment (giordano) [inline L/LibCURL/build_tarballs.jl:51]:
```suggestion
    ExecutableProduct("curl", :curl),
```

File: L/LibCURL/build_tarballs.jl [hunk 2]
```diff
@@ -3,16 +3,12 @@
...
+version = v"7.71.1"
 
 # Collection of sources required to build LibCURL
 sources = [
     ArchiveSource("https://curl.haxx.se/download/curl-$(version).tar.gz", 
-    "ca2feeb8ef13368ce5d5e5849a5fd5e2dd4755fecf7d8f0cc94000a4206fb8e7"), 
-    FileSource("https://curl.haxx.se/ca/cacert-$cacert_version.pem", 
-    "adf770dfd574a0d6026bfaa270cb6879b063957177a991d453ff1d302c02081f",
-    filename="cacert.pem")
+    "59ef1f73070de67b87032c72ee6037cedae71dcb1d7ef2d7f59487704aec069d")
```
Comment (giordano) [inline L/LibCURL/build_tarballs.jl:11]:
```suggestion
    "59ef1f73070de67b87032c72ee6037cedae71dcb1d7ef2d7f59487704aec069d"),
```

**Pull request merged**